src/main/ast-to-doc.js

```diff
@@ -78,7 +78,7 @@ function printAstToDoc(ast, options, alignmentSize = 0) {
     // Add a hardline to make the indents take effect
     // It should be removed in index.js format()
     doc = addAlignmentToDoc(
-      docUtils.removeLines(concat([hardline, doc])),
```

Comment:
Yeah, it was collapsed before this PR:

**Prettier 1.15.3**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAzArlMMCW0AE6A5gDYCeAFAJT7AA6U++kUAzjPgCY7YDqOMABYBlOADc4AJwCGJAKKxJOOK3wBeWgyZMA1nDJJtR7XRBjZ6OKYA0W7dKgQhUgNL7DTUw6eCp+cySWNnZMqDiS7ABy6AC2AEZSHgCMtozarHAsnNHxifgATHYAvgDcDEUg1iAQAA640KzIoNKSkhAA7gAKLQiNKNJiEDiclSBxMmB6MMI10mA4UETIMJKWVYIwMSS8ggIqs2BwomwCOGICZMjgrI1VCxmSMJ0yRDHSyKiyGVUAVqwAHgAhCZTYTSGJwAAyCzgHy+cF+AOEC1IcAAiugnLCkJ8SN8QLMIlIrnFpHEyCRoKMakpYPxOEJkAAOAAMVRpEAyvBkNSuNJUUgko0kcAAjuhwnBntJXu8cfCqhkYjhlqsESBWCiSOjMfA4Xj1TAyfTGUh8lUVtIcCQUQBhCAxN5XRxQWFVdAZAAqZL6uPxMkWRyNj2QAHYAJxVANEOAKEZIJIAVgALEUikA)
```sh
--parser babylon
--range-end 154
--range-start 79
```

**Input:**
```jsx
function ugly() {
  const dictWithSeveralEntries = {
    key:          "value",
    anotherKey:   "another value",
    firstNumber:  1,
    secondNumber: 2
  };
}
```

**Output:**
```jsx
function ugly() {
  const dictWithSeveralEntries = { key: "value", anotherKey: "another value", firstNumber: 1, secondNumber: 2 };
}
```

Reply:
Oh, I see, thanks for the explanation! Since there's no broken tests, I think this is fine to merge. If we get bug reports that are related to deleting the `removeLines` call, we can figure things out then.